src/base/iconprovider.cpp

```diff
@@ -35,7 +36,9 @@ IconProvider::IconProvider(QObject *parent)
 {
 }
 
-IconProvider::~IconProvider() {}
+IconProvider::~IconProvider()
```

Comment:
I see what you mean though - c++11 `= default`. Still catching up on c++11 - much of this stuff wasn't around last time I used c++ (~10 years ago). And it's been over 20 years since I hung around on comp.std.c++ :)